### YamlMime:ManagedReference
items:
- uid: System.Data.SqlTypes.SqlTypeException
  id: SqlTypeException
  children:
  - System.Data.SqlTypes.SqlTypeException.#ctor
  - System.Data.SqlTypes.SqlTypeException.#ctor(System.String)
  - System.Data.SqlTypes.SqlTypeException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Data.SqlTypes.SqlTypeException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: SqlTypeException
  nameWithType: SqlTypeException
  fullName: System.Data.SqlTypes.SqlTypeException
  type: Class
  summary: "A kiinduló kivétel osztály a <xref href=&quot;System.Data.SqlTypes&quot;> </xref>."
  syntax:
    content: 'public class SqlTypeException : SystemException'
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Data.SqlTypes.SqlTypeException.#ctor
  id: '#ctor'
  parent: System.Data.SqlTypes.SqlTypeException
  langs:
  - csharp
  name: SqlTypeException()
  nameWithType: SqlTypeException.SqlTypeException()
  fullName: System.Data.SqlTypes.SqlTypeException.SqlTypeException()
  type: Constructor
  assemblies:
  - System.Data
  namespace: System.Data.SqlTypes
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.SqlTypes.SqlTypeException&quot;> </xref> osztály."
  syntax:
    content: public SqlTypeException ();
    parameters: []
  overload: System.Data.SqlTypes.SqlTypeException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Data.SqlTypes.SqlTypeException
  langs:
  - csharp
  name: SqlTypeException(String)
  nameWithType: SqlTypeException.SqlTypeException(String)
  fullName: System.Data.SqlTypes.SqlTypeException.SqlTypeException(String)
  type: Constructor
  assemblies:
  - System.Data
  namespace: System.Data.SqlTypes
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.SqlTypes.SqlTypeException&quot;> </xref> osztály egy adott hiba miatt."
  syntax:
    content: public SqlTypeException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A hibaüzenet, amely ismerteti a kivétel az az oka."
  overload: System.Data.SqlTypes.SqlTypeException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Data.SqlTypes.SqlTypeException
  langs:
  - csharp
  name: SqlTypeException(SerializationInfo,StreamingContext)
  nameWithType: SqlTypeException.SqlTypeException(SerializationInfo,StreamingContext)
  fullName: System.Data.SqlTypes.SqlTypeException.SqlTypeException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Data
  namespace: System.Data.SqlTypes
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.SqlTypes.SqlTypeException&quot;> </xref> a szerializált adatok osztályt."
  remarks: "Ez a konstruktor nevezik meglévő kölcsönökre az adatfolyam továbbított kivétel objektum deszerializálása közben. További információkért lásd: [XML és a SOAP-szerializálási](~/add/includes/ajax-current-ext-md.md)."
  syntax:
    content: protected SqlTypeException (System.Runtime.Serialization.SerializationInfo si, System.Runtime.Serialization.StreamingContext sc);
    parameters:
    - id: si
      type: System.Runtime.Serialization.SerializationInfo
      description: "A szerializált objektum adatokat tartalmazó objektum."
    - id: sc
      type: System.Runtime.Serialization.StreamingContext
      description: "A cél- és környezeti információt."
  overload: System.Data.SqlTypes.SqlTypeException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Data.SqlTypes.SqlTypeException
  langs:
  - csharp
  name: SqlTypeException(String,Exception)
  nameWithType: SqlTypeException.SqlTypeException(String,Exception)
  fullName: System.Data.SqlTypes.SqlTypeException.SqlTypeException(String,Exception)
  type: Constructor
  assemblies:
  - System.Data
  namespace: System.Data.SqlTypes
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.SqlTypes.SqlTypeException&quot;> </xref> megadott hibaüzenet, és hivatkozni kell a belső kivételben találhatók okozza-e ezt a kivételt, a osztályt."
  syntax:
    content: public SqlTypeException (string message, Exception e);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel leíró üzenet. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított."
    - id: e
      type: System.Exception
      description: "A kivétellel, hogy a jelenlegi kivétel okozza. Ha a `innerException` paraméter nem <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a jelenlegi kivétel a következik be egy <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blokk, amely kezeli a belső kivételben találhatók."
  overload: System.Data.SqlTypes.SqlTypeException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Data.SqlTypes.SqlTypeException.#ctor
  parent: System.Data.SqlTypes.SqlTypeException
  isExternal: false
  name: SqlTypeException()
  nameWithType: SqlTypeException.SqlTypeException()
  fullName: System.Data.SqlTypes.SqlTypeException.SqlTypeException()
- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.String)
  parent: System.Data.SqlTypes.SqlTypeException
  isExternal: false
  name: SqlTypeException(String)
  nameWithType: SqlTypeException.SqlTypeException(String)
  fullName: System.Data.SqlTypes.SqlTypeException.SqlTypeException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Data.SqlTypes.SqlTypeException
  isExternal: false
  name: SqlTypeException(SerializationInfo,StreamingContext)
  nameWithType: SqlTypeException.SqlTypeException(SerializationInfo,StreamingContext)
  fullName: System.Data.SqlTypes.SqlTypeException.SqlTypeException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.String,System.Exception)
  parent: System.Data.SqlTypes.SqlTypeException
  isExternal: false
  name: SqlTypeException(String,Exception)
  nameWithType: SqlTypeException.SqlTypeException(String,Exception)
  fullName: System.Data.SqlTypes.SqlTypeException.SqlTypeException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Data.SqlTypes.SqlTypeException.#ctor*
  parent: System.Data.SqlTypes.SqlTypeException
  isExternal: false
  name: SqlTypeException
  nameWithType: SqlTypeException.SqlTypeException
